tizen/src/workmanager_tizen_plugin.cc

```diff
@@ -0,0 +1,347 @@
+                                 existing_work_policy, frequency_seconds,
+                                 initial_delay_seconds, constraints_config,
+                                 backoff_policy_config, out_of_quota_policy,
+                                 tag, payload),
+                    std::move(result));
+                return;
+            }
+            result->Error("WRONG_ARGS",
+                          "Wrong argument for registerPeriodicTask");
+            return;
+        } else if (call.method_name() ==
+                   constants::methods::kCancelTaskByUniqueName) {
+            const auto& args = *call.arguments();
+            if (std::holds_alternative<flutter::EncodableMap>(args)) {
+                flutter::EncodableMap map =
+                    std::get<flutter::EncodableMap>(args);
+                auto value = GetOrNullFromEncodableMap<std::string>(
+                    &map, constants::keys::kCancelTaskUniqueNameKey);
+                if (value.has_value()) {
+                    HandleCancelByName(CancelByNameTask(value.value()),
+                                              std::move(result));
+                    return;
+                }
+            }
+
+            result->Error("WRONG_ARGS",
+                          "Wrong argument for cancelTaskByUniqueName");
+            return;
+        } else if (call.method_name() == constants::methods::kCancelTaskByTag) {
+            const auto& args = *call.arguments();
+            if (std::holds_alternative<flutter::EncodableMap>(args)) {
+                flutter::EncodableMap map =
+                    std::get<flutter::EncodableMap>(args);
+                auto value = GetOrNullFromEncodableMap<std::string>(
+                    &map, constants::keys::kCancelTaskTagKey);
+                if (value.has_value()) {
+                    HandleCancelByTag(CancelByTagTask(value.value()),
+                                       std::move(result));
+                }
+            }
+            result->Error("WRONG_ARGS", "Wrong argument for cancelTaskBytag");
+            return;
+        } else if (call.method_name() == constants::methods::kCancelAllTasks) {
+            HandleCancelAll(std::move(result));
+        }
+
+        result->NotImplemented();
+    }
+
+    void HandleInitializeTask(const InitializeTask& call,
+                           std::unique_ptr<FlMethodResult> result) {
+        preference_set_int(constants::keys::kDispatcherHandleKey,
+                           call.callback_dispathcer_handler_key);
+
+        result->Success();
+    }
+
+    void HandleOneOffTask(const OneoffTask& call, std::unique_ptr<FlMethodResult> result) {
+        bool initialized = false;
+        preference_is_existing(constants::keys::kDispatcherHandleKey,
+                               &initialized);
+        if (!initialized) {
+            result->Error("1", constants::kNotInitializedErrMsg,
+                          flutter::EncodableValue(""));
```

Comment:
TODO?

Reply:
Fixed it by replace detailed message into nullptr like android plugin. [Android plugin error detail code](https://github.com/fluttercommunity/flutter_workmanager/blob/44ba91f2ab527688bd2c4fab270caaa2766ec707/android/src/main/kotlin/be/tramckrijte/workmanager/WorkmanagerCallHandler.kt#L87C17-L87C21)